Makefile

```diff
@@ -43,6 +43,13 @@ privacyidea:
 	#(cd ${BUILDDIR_PI}; git checkout v${GIT_VERSION})
 	#(cd ${BUILDDIR_PI}; git submodule init; git submodule update --recursive --remote)
 	(cd ${BUILDDIR_PI}; rm -fr tests)
+	# --- Build new WebUI (requires npm/node) ---
+	@command -v npm >/dev/null 2>&1 || { echo >&2 "ERROR: npm not found. Please install nodejs & npm for the build host."; exit 1; }
```

Comment:
Do we have a "builder"-definition somewhere?
I wanted to use the `Build-Depends:` from [`debian_privacyidea/control`](../../blob/master/debian_privacyidea/control) so we can use something like `mk-build-deps` to set up the builder: https://serverfault.com/questions/127625/given-a-debian-source-package-how-do-i-install-the-build-deps